examples/data-objects/primitives/src/main.tsx

```diff
@@ -43,7 +43,7 @@ export class PrimitivesCollection extends DataObject implements IFluidHTMLView {
     public render(div: HTMLElement) {
         const mapCreate = (name: string) => SharedMap.create(this.runtime, name);
         const mapListen = (listener: (changed: IDirectoryValueChanged) => void) => {
```

Comment:
The event arg type is `IValueChanged` for the `"containedValueChanged"` event:

```suggestion
        const mapListen = (listener: (changed: IValueChanged) => void) => {
```

Reply:
@ChumpChief Made the proposed changes.